en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,455 @@
+# Redis Connector Example
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs and geospatial indexes with radius queries.
+
+This connector enables developers to use an external Redis server as a cache or a database in the WSO2 EI mediation logic.
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI Redis Connector and access the Redis server, using Redis commands.
+
+The user sends the request for invoke to an API to get WSO2 stock details. this REST call will get converted into a SOAP message and sent to the back-end service. While the response from the backend service is converted back to json and sent back to the API caller, WSO2 EI will extract stock volume details from the response and use **/getstockquote** operation to store it into configured Redis server.
```

Comment:
Shall we update as WSO2 EI will extract stock volume details from the response store it into configured Redis server?

Reply:
Updated.